packages/plugin-commands-init/src/init.ts

```diff
@@ -40,8 +40,11 @@ export async function handler (
   if (fs.existsSync(manifestPath)) {
     throw new PnpmError('PACKAGE_JSON_EXISTS', 'package.json already exists')
   }
+  const response = await fetch('https://registry.npmjs.org/pnpm/latest')
```

Comment:
This isn't correct because it doesn't use the network and registry configurations.